htdocs/index.php

```diff
@@ -22,6 +22,13 @@
 // to be done before NDB_Client starts the PHP session.)
 session_cache_limiter("");
 
+// PHP documentation says this should always be enabled for session security.
+// PHP documentation says this is disabled by default.
+// Explicitly enable it.
+// phpcs:ignore
```

Comment:
I don't think there's any choice but to ignore the next line since the URL is so long